### DISCUSSION TITLE
Import Plex Users -> "There are no Plex users to import."

### DISCUSSION THREAD

**Discussion by nayab9** [sct/overseerr#2875] [Help] [answered]
I must be missing something trivial here... if anybody can point out the obvious I would greatly appreciate it!

I have a plex server which I log into via plex.tv with my plex account no issues there.
My plex server has a several "managed users" who access the media, no issues there.
I run Tautulli, it keeps track of all the managed users information, no issues there.

Would love to use overseerr - but after logging into overseerr (via plex oauth), I only have access to the admin user.
When I click on "Import Plex Users", it says there are no users (picture attached).

I must be doing something incorrectly here, but not sure what - doesn't seem to be much google-able info about this...

![image](https://user-images.githubusercontent.com/1916171/182007764-c2fd0064-4813-4912-8177-8dda84a16675.png)

**Comment by TheCatLady** [comment, marked as answer]
Managed users do not have their own Plex accounts/logins, and thus cannot be imported. You can create full Plex accounts for your users and then invite them to your Plex Home.

**Reply by nayab9** [reply]
Thanks @TheCatLady!

That clears it up. All my users are managed users - so that's why they are not seen by the importer.

Is this limitation in overseerr a backlog item? Based on the way tautulli is able to keep track of all the managed users, I imagine nothing is stopping this from being implemented in overseerr other than spare dev cycles? If so, I will start taking a look.

**Reply by TheCatLady** [reply]
The limitation is that managed users do not have their own Plex accounts and therefore don't have their own logins. Local user functionality exists in Overseerr for this reason.

**Reply by nayab9** [reply]
Can the overseerr 'local user' somehow plug into a plex 'managed user'? Otherwise it wouldn't be giving them suggestions based on their media consumption behavior (unless I'm not understanding something)..

**Reply by TheCatLady** [reply]
No. Overseerr doesn't give suggestions based on media consumption behavior regardless of whether a Plex account is linked.

**Reply by nayab9** [reply]
Well that wraps it up!

I'm a total dunce. Thanks so much for the replies.
